Description: Use dpkg-architecture in cups-config to make it architecture-independant
Bug-Debian: https://bugs.debian.org/741519
Author: Didier Raboud <odyx@debian.org>
Last-Update: 2014-03-24
--- a/cups-config.in
+++ b/cups-config.in
@@ -22,8 +22,8 @@
 exec_prefix=@exec_prefix@
 bindir=@bindir@
 includedir=@includedir@
-libdir=@libdir@
-imagelibdir=@libdir@
+libdir=${prefix}/lib/$(dpkg-architecture -qDEB_HOST_ARCH)
+imagelibdir=${prefix}/lib/$(dpkg-architecture -qDEB_HOST_ARCH)
 datarootdir=@datadir@
 datadir=@datadir@
 sysconfdir=@sysconfdir@
